addons/binding/org.openhab.binding.sleepiq/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,37 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: SleepIQ Binding
+Bundle-SymbolicName: org.openhab.binding.sleepiq;singleton:=true
+Bundle-Vendor: openHAB
+Bundle-Version: 2.1.0.qualifier
+Bundle-RequiredExecutionEnvironment: JavaSE-1.8
+Bundle-ClassPath: .,
+ lib/sleepiq-api.jar
+Import-Package: 
+ com.eclipsesource.jaxrs.provider.gson,
```

Comment:
@syphr42 Please see https://github.com/eclipse/smarthome/pull/3682 - ESH does not at all use the com.eclipsesource.jaxrs.provider.gson package anymore and thus I would actually want to avoid including it in openHAB as well. As the GsonProvider is indeed just a few lines of code: Could you simply implement this in your sleepiq-api library and such remove this dependency here completely?